### ISSUE TITLE
Should the macOS autostart doc be updated?

### ISSUE THREAD

**Issue by xgt001** [home-assistant/home-assistant.github.io#12947]
**TL:DR**
hass service fails to start on macOS Catalina 10.15.4 on a clean venv installation due to missing `autoconf` blocking pytradfri to be installed. Should the [doc](https://www.home-assistant.io/docs/autostart/macos/) be updated to include `brew install automake` with a note about this?

Long sequence of events below if anyone is interested.
I am happy to update the docs if you folks think its okay for this step to be included.

Logs:
```
2020-04-13 23:37:38 ERROR (MainThread) [homeassistant.core] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/Users/ganesh.hegde/hobbies/homeassistant/homeassistant/lib/python3.7/site-packages/homeassistant/data_entry_flow.py", line 100, in async_init
    flow = await self.async_create_flow(handler, context=context, data=data)
  File "/Users/ganesh.hegde/hobbies/homeassistant/homeassistant/lib/python3.7/site-packages/homeassistant/config_entries.py", line 501, in async_create_flow
    await async_process_deps_reqs(self.hass, self._hass_config, integration)
  File "/Users/ganesh.hegde/hobbies/homeassistant/homeassistant/lib/python3.7/site-packages/homeassistant/setup.py", line 291, in async_process_deps_reqs
    hass, integration.domain
  File "/Users/ganesh.hegde/hobbies/homeassistant/homeassistant/lib/python3.7/site-packages/homeassistant/requirements.py", line 55, in async_get_integration_with_requirements
    hass, integration.domain, integration.requirements
  File "/Users/ganesh.hegde/hobbies/homeassistant/homeassistant/lib/python3.7/site-packages/homeassistant/requirements.py", line 105, in async_process_requirements
    raise RequirementsNotFound(name, [req])
homeassistant.requirements.RequirementsNotFound: Requirements for tradfri not found: ['pytradfri[async]==6.4.0'].
```

Which upon some further digging revealed to be missing `automake` in macOS which was not letting pytradfri to be installed.
```
pip3 install pytradfri[async]==6.4.0                                                                                                                               23:38:11
Collecting pytradfri[async]==6.4.0
Requirement already satisfied: aiocoap==0.4a1; extra == "async" in ./homeassistant/lib/python3.7/site-packages (from pytradfri[async]==6.4.0) (0.4a1)
Collecting DTLSSocket==0.1.7; extra == "async" (from pytradfri[async]==6.4.0)
  Using cached https://files.pythonhosted.org/packages/d3/4d/7bb7699aa4ba47e0b4287165ebaabca13461e17fafe7d2d99e3bfb2c3299/DTLSSocket-0.1.7.tar.gz
Requirement already satisfied: Cython in ./homeassistant/lib/python3.7/site-packages (from DTLSSocket==0.1.7; extra == "async"->pytradfri[async]==6.4.0) (0.29.16)
Installing collected packages: DTLSSocket, pytradfri
  Running setup.py install for DTLSSocket ... error
    Complete output from command /Users/ganesh.hegde/hobbies/homeassistant/homeassistant/bin/python3 -u -c "import setuptools, tokenize;__file__='/private/var/folders/ty/q5gmv1691xx_gzt4kc5zjtysbtwnyl/T/pip-install-kfuuc5iy/DTLSSocket/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /private/var/folders/ty/q5gmv1691xx_gzt4kc5zjtysbtwnyl/T/pip-record-_gotxoo9/install-record.txt --single-version-externally-managed --compile --install-headers /Users/ganesh.hegde/hobbies/homeassistant/homeassistant/include/site/python3.7/DTLSSocket:
    running install
    running build
    running build_py
    creating build
    creating build/lib.macosx-10.14-x86_64-3.7
    creating build/lib.macosx-10.14-x86_64-3.7/DTLSSocket
    copying DTLSSocket/__init__.py -> build/lib.macosx-10.14-x86_64-3.7/DTLSSocket
    copying DTLSSocket/DTLSSocket.py -> build/lib.macosx-10.14-x86_64-3.7/DTLSSocket
    running build_ext
    Running: autoconf
    error: [Errno 2] No such file or directory: 'autoconf': 'autoconf'
```

`brew install automake` fixed it for me and started the `hass` service without any fuss.

**Comment by fabaff** [comment]
This will also affect regular macOS installations. The requirements are installed on demand there as well.

Would be nice if you could update the docs. Thanks.

**Comment by xgt001** [comment]
Thanks @fabaff , will do it today :)

**Comment by xgt001** [comment]
hi @fabaff , please take a look at #13006  :)
